docs/book/src/SUMMARY.md

```diff
@@ -56,5 +56,6 @@
     - [Ports](./reference/ports.md)
     - [Code of Conduct](./code-of-conduct.md)
     - [Contributing](./CONTRIBUTING.md)
```

Comment:
Could we remove the `Reviews` section from `CONTRIBUTING.md` if we have this one?
It's stating a subset of the information declared in [`(from Code Review Developer Guide - Google)`](https://google.github.io/eng-practices/review/)

This way new contributors are directed to a single source of information regarding reviews.

Reply:
SGTM 👍

Reply:
Uhmm, the original idea from @vincepri was to have a generic chapter about review in CONTRIBUTING.md and all the CAPI specific bits in REVIEWING.md, but I leave to him the final call about this.

Also, FYI, https://github.com/kubernetes-sigs/cluster-api/pull/3349 is making some cometic changes to the review chapter in CONTRIBUTING.md

Reply:
Ahh..ok. I guess if we want to have a generic Review paragraph in CONTRIBUTING, could we at least include a link to REVIEWING.md for CAPI specific review process?

Reply:
This should be already the case

Reply:
The contributing guide is usually what folks lean towards to and it's the mostly widely known